requirements-dev.txt

```diff
@@ -1,2 +1,2 @@
 autopep8
-localstack==0.9.6
+localstack[full]
```

Comment:
What does [full] mean? I couldn't find much from a quick google search and thought it'd be more productive to just ask here.

Reply:
I've also never seen this before, but it was recommended in the `localstack` README. My understanding is that it let's package managers specify "versions" of the same release of their package. We're installing the "full" version. See: https://setuptools.readthedocs.io/en/latest/pkg_resources.html#requirement-objects.